### DISCUSSION TITLE
How to get size of each tile in pixels onResizeFinished

### DISCUSSION THREAD

**Discussion by khushboopareek-nice** [DevbookHQ/splitter#24] [Q&A] [answered]
I am using ReactSplit and I have given initialsizes in percentage as referred in examples. and I want the sizes of each tile once resize is finished. But it is not working. 
![image](https://user-images.githubusercontent.com/93115310/205843632-bc2d07c4-2963-4d24-bea4-5cc28e6315b0.png)
![image](https://user-images.githubusercontent.com/93115310/205843836-aa9a16a7-5b16-4cfa-848a-a71cec04c623.png)

**Comment by mlejva** [comment, marked as answer]
You have to provide a link to a live example where I can reproduce the issue if you want me to look into it

**Reply by khushboopareek-nice** [reply]
I added my code here- https://codesandbox.io/s/quizzical-snowflake-0q4p85?file=/src/App.js 
I am able to get the sizes in % but not in pixels.

**Reply by mlejva** [reply]
Thanks for the example.
There isn't a way to get size of tiles directly in pixels with splitter's props. It's left on users by calculating it based on the size of the splitter's parent element.
